user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md

```diff
@@ -34,6 +34,9 @@ For a self-hosted DataMiner System, follow the steps below to set up STaaS.
 
 1. [Upgrade your DataMiner System](xref:Upgrading_a_DataMiner_Agent) to version 10.4.0 [CU0] or higher.
 
+   > [!NOTE]
+   > To be able to use non-indexed Logger Tables, version 10.4.0 [CU5] or higher must be installed.
```

Comment:
Which version is required for people on the feature release track?

Reply:
10.4.8 CU0

Reply:
```suggestion
   > To be able to use non-indexed logger tables, upgrade to DataMiner version 10.4.0 [CU5]/10.4.8 or higher.
```